app/models/maintenance_stats/queries/github/repository_contributor_stats_query.rb

```diff
@@ -0,0 +1,25 @@
+module MaintenanceStats
+  module Queries
+    module Github
+      class RepositoryContributorStatsQuery < BaseQuery
+        @@valid_params = [:full_name]
+        @@required_params = [:full_name]
+        @@retry_amount = 4
+        def self.client_type
+          :v3
+        end
+        def query(params: {}, count: 0)
+          raise Octokit::Error.new "Didn't get a response after #{@@retry_amount} attempts" if params[:count].present? && params[:count] >= @@retry_amount
+          validate_params(params)
+
+          resp = @client.contributor_stats(params[:full_name])
+          if @client.last_response.status == 202
+            sleep 1
```

Comment:
4 seconds of total possible sleep time sounds like a lot, although I'm not sure what this is waiting on?

Reply:
This request to GitHub will often return a `202` instead of the actual data to let GitHub process the information needed for the request. This is there to retry the call after a short wait, not ideal. It would be better if this was handled in the Octokit client.

Reply:
Maybe use a simple increasing backoff?